recipes/debugging/vs-code-debug.md

```diff
@@ -1,40 +1,55 @@
-# VS Code Amber Debug
+# Debugging Crystal Projects
```

Comment:
Since the guide is strongly focused on VS Code, I think this title is too generic.